ext/openssl/openssl.c

```diff
@@ -935,7 +944,7 @@ static time_t asn1_time_to_time_t(ASN1_UTCTIME * timestr) /* {{{ */
 		return (time_t)-1;
 	}
 
-	strbuf = estrdup((char *)ASN1_STRING_data(timestr));
+	strbuf = estrdup((char *)ASN1_STRING_get0_data(timestr));
```

Comment:
I know that it was already there but when you are in it, pls remove that pointless cast to `char *`